### PR TITLE
[disasters] event map updates

### PR DESCRIPTION
**Pull request by chejennifer**
- add tooltip that shows place name when hovering over a region
- make dot size match severity
Screenshot 2023-01-18 at 2 59 44 PM

### CODE REVIEW

File: static/js/components/tiles/disaster_event_map_tile.tsx [hunk 1]
```diff
@@ -384,24 +406,27 @@ export function DisasterEventMapTile(
...
+      const pointsLayer = addMapPoints(
+        props.id,
+        mapPointsData.points,
+        mapPointsData.values,
+        projection,
+        (point: DisasterEventPoint) => {
+          return props.eventTypeSpec[point.disasterType].color;
+        },
+        undefined,
+        0.8
```
Comment (beets) [inline static/js/components/tiles/disaster_event_map_tile.tsx:419]: use a const for this

File: static/js/components/tiles/disaster_event_map_tile.tsx [hunk 2]
```diff
@@ -332,10 +338,27 @@ export function DisasterEventMapTile(
         Object.values(disasterEventData.provenanceInfo).forEach((provInfo) => {
           sources.add(provInfo.provenanceUrl);
         });
+        const mapPointsData = {};
```
Comment (beets) [inline static/js/components/tiles/disaster_event_map_tile.tsx:341]: pending time, would be nice to split this up into smaller functions. in the meantime, add some comments?

**Review by beets — APPROVED** [review]
looks good!

File: static/js/components/tiles/disaster_event_map_tile.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/components/tiles/disaster_event_map_tile.tsx]: done

File: static/js/components/tiles/disaster_event_map_tile.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/components/tiles/disaster_event_map_tile.tsx]: done, created a utils function for this

**Pull request merged**